src/d/actor/d_a_obj_barrel.cpp

```diff
@@ -4,226 +4,758 @@
 //
 
 #include "d/actor/d_a_obj_barrel.h"
+#include "d/actor/d_a_obj_eff.h"
+#include "d/res/res_ktaru_01.h"
+#include "f_op/f_op_actor_mng.h"
+#include "f_op/f_op_kankyo_mng.h"
 #include "d/d_procname.h"
+#include "d/d_com_inf_game.h"
+
+#include "weak_data_1811.h" // IWYU pragma: keep
+
+const char daObjBarrel::Act_c::M_arcname[] = "Ktaru_01";
+const float daObjBarrel::Act_c::l_s_radius = 45.0f;
+const float daObjBarrel::Act_c::l_l_radius = 50.0f;
+const float daObjBarrel::Act_c::l_gnd_fric = 0.1f; // TODO use
+const short daObjBarrel::Act_c::l_gnd_deg = 0xf; // TODO ??
+const float daObjBarrel::Act_c::l_viscous_resist = 0.006f;
+const float daObjBarrel::Act_c::l_inert_resist = 0.001f; // TODO use more?
```

Comment:
Old comments?

Reply:
good catch, I'll remove these
I will call out that I'm not sure if I've over- or under-used these constants since the generated assembly doesn't change whether I use them or not. I've given it my best guess (although `l_gnd_deg` is still unused, maybe one of the inlines uses it)